languages.toml

```diff
@@ -2928,3 +2928,16 @@ language-servers = [ "templ" ]
 [[grammar]]
 name = "templ"
 source = { git = "https://github.com/vrischmann/tree-sitter-templ", rev = "ea56ac0655243490a4929a988f4eaa91dfccc995" }
+
+[[language]]
+name = "dbml"
+scope = "source.dbml"
+injection-regex = "dbml"
+file-types = ["dbml"]
+roots = []
```

Comment:
This `roots` key can be omitted if you merge/rebase on the latest master

Reply:
line is removed :)